libs/user-facing-errors/src/lib.rs

```diff
@@ -95,7 +95,13 @@ impl Error {
     /// Construct a new UnknownError from a `PanicInfo` in a panic hook. `UnknownError`s created
     /// with this constructor will have a proper, useful backtrace.
     pub fn new_in_panic_hook(panic_info: &std::panic::PanicInfo<'_>) -> Self {
-        let message = Self::extract_panic_message(panic_info.payload()).unwrap_or_else(|| "<unknown panic>".to_owned());
```

Comment:
is `extract_panic_message()` completely unused now? If so we could delete it.

Reply:
look a few lines down @tomhoule  :D